src/main/java/core/basesyntax/report/CsvFileWriter.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.report;
+
+import core.basesyntax.exceptions.FileWasNotWrittenException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class CsvFileWriter {
+    private final String filePath;
+
+    public CsvFileWriter(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void writeReport(Map<String, Integer> reportData) {
+        try (FileWriter writer = new FileWriter(filePath)) {
+            StringBuilder stringBuilder = new StringBuilder();
```

Comment:
bad naming, don't name after container type, name after what it stores, will make code more readable

Reply:
not fixed - bad naming, don't name after container type, name after what it stores, will make code more readable
Don't name variables like String string, List list etc. Naming is an opportunity to make code more understandable for a person who will read it for the first time. Remember, that developers spend more time reading code, than actually writing it, by far